Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -289,6 +289,19 @@ public partial class Scheduler : IPipScheduler, IPipExecutionEnvironment, IFileC
         private int m_maxUnresponsivenessFactor = 0;
         private DateTime m_statusLastCollected = DateTime.MaxValue;
 
+        /// <summary>
+        /// Output file contents which affected by the source change of this build. 
+        /// </summary>
+        /// <remarks>
+        /// This list should be initialized with the source change list provided by GBR.

```

Comment:
>GBR [](start = 84, length = 3)

Not necessarily `GBR`. I think what you should say is it is initialized by the input change list provided from the configuration option that I introduced last time.